tests/scripts/run-corefx-tests.py

```diff
@@ -260,17 +284,44 @@ def main(args):
 
     config_args = '-Release -os:%s -buildArch:%s' % (clr_os, arch)
 
-    # Run the primary (non-test) corefx build
+    # Run the primary (non-test) corefx build. We previously passed the argument:
+    #
+    #    /p:CoreCLROverridePath=<path-to-core_root>
+    #
+    # which causes the corefx build to overwrite its built runtime with the binaries from
+    # the coreclr build. However, this often causes build failures when breaking changes are
+    # in progress (e.g., a breaking change is made in coreclr that has not yet had compensating
+    # changes made in the corefx repo). Instead, build corefx normally. This should always work
+    # since corefx is protected by a CI testing system. Then, overwrite the built corefx
+    # runtime with the runtime built in the coreclr build. The result will be that perhaps
+    # some, hopefully few, corefx tests will fail, but the builds will never fail.
 
     command = ' '.join(('build.cmd' if Is_windows else './build.sh',
-                        config_args,
-                        '-- /p:CoreCLROverridePath=%s' % core_root))
+                        config_args))
 
     log(command)
     returncode = 0 if testing else os.system(command)
     if returncode != 0:
         sys.exit(1)
 
+    # Override the built corefx runtime (which it picked up by copying from packages determined
+    # by its dependencies.props file). Note that we always build Release corefx.
+    # We must copy all files, not just the files that already exist in the corefx runtime
+    # directory. This is required so we copy over all altjit compilers.
+    # TODO: it might be cleaner to encapsulate the knowledge of how to do this in the
+    # corefx msbuild files somewhere.
+
+    fx_runtime = os.path.join(fx_root,
```

Comment:
I don't think this is a good idea. It will likely break whenever we change the layout of corefx output. We don't want to be responsible for updating random scripts like this that are taking a hard-coded dependency on the output of corefx repo. 

If the options of CoreCLROverridePath isn't doing what you need it to then we should fix that or provide another public hook instead of depending on these paths here.

Reply:
Agreed; hence https://github.com/dotnet/corefx/issues/26944